src/ogc-api/endpoint.ts

```diff
@@ -245,4 +245,69 @@ export default class OgcApiEndpoint {
       })
       .then(fetchDocument<OgcApiCollectionItem>);
   }
+
+  /**
+   * Asynchronously retrieves a URL for the items of a specified collection, with optional query parameters.
+   * @param collectionId - The unique identifier for the collection.
+   * @param options - An object containing optional parameters:
+   *  - query: Additional query parameters to be included in the URL.
+   *  - outputFormat: The MIME type for the output format. Default is 'json'.
+   *  - maxFeatures: The maximum number of features to include.
+   *  - skipGeometry: Flag to skip geometry data in the response.
+   *  - extent: Bounding box to limit the features.
+   *  - offset: Pagination offset for the returned results.
+   *  - outputCrs: Coordinate Reference System code for the output.
+   *  - extentCrs: Coordinate Reference System code for the bounding box.
+   *  - properties: An array of property names to be returned.
+   * @returns A promise that resolves to the URL as a string or rejects if an error occurs.
+   */
+  getCollectionItemsUrl(
+    collectionId: string,
+    options: {
+      query?: string;
+      outputFormat?: string;
+      maxFeatures?: number;
+      skipGeometry?: boolean;
+      offset?: number;
+      outputCrs?: string;
+      extent?: number[];
+      extentCrs?: string;
+      properties?: string[];
+    } = {} // Make options optional and default to an empty object
+  ): Promise<string> {
+    return this.getCollectionDocument(collectionId)
+      .then((collectionDoc) => {
+        const baseUrl = this.baseUrl || '';
+        const itemsLink = getLinkUrl(collectionDoc, 'items', baseUrl);
+        const url = new URL(itemsLink);
+
+        // Set the format to JSON if not specified
+        const format = options.outputFormat || 'json';
+        url.searchParams.set('f', format);
+
+        // Inline conditional statements for other options
+        if (options.query !== undefined)
+          url.search += (url.search ? '&' : '') + options.query;
+        if (options.maxFeatures !== undefined)
+          url.searchParams.set('limit', options.maxFeatures.toString());
+        if (options.offset !== undefined)
+          url.searchParams.set('offset', options.offset.toString());
+        if (options.skipGeometry !== undefined)
+          url.searchParams.set('skipGeometry', options.skipGeometry.toString());
+        if (options.outputCrs !== undefined)
+          url.searchParams.set('outputCrs', options.outputCrs);
+        if (options.extent !== undefined && options.extent.length === 4)
+          url.searchParams.set('bbox', options.extent.join(','));
+        if (options.extentCrs !== undefined)
+          url.searchParams.set('bbox-crs', options.extentCrs);
+        if (options.properties !== undefined && options.properties.length > 0)
+          url.searchParams.set('properties', options.properties.join(','));
```

Comment:
I don't think this is supported by OGC API, do you? If not we should leave it out for now